agent/src/com/jonnyzzz/teamcity/virtual/run/VMRunnerContext.java

```diff
@@ -68,14 +69,20 @@ public File getAgentTempDirectory() {
 
   @NotNull
   public String getCheckoutMountPoint() {
+    String defaultMountPoint;
```

Comment:
a `final` declaration is nicer here (can be also combined with ternary operator)